base/applications/regedit/clb/lang/tr-TR.rc

```diff
@@ -19,7 +19,7 @@ BEGIN
    CHECKBOX "&Esnek Sütunlar", 1715, 79, 50, 66, 10, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
    GROUPBOX "Temel Biçimler", -1, 6, 80, 158, 34
    CHECKBOX "Gö&rünür", 1701, 10, 92, 34, 10, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
-   CHECKBOX "Dev&re Dışı", 1702, 10, 102, 41, 10, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
+   CHECKBOX "Edi&lgin", 1702, 10, 102, 41, 10, BS_AUTOCHECKBOX | WS_GROUP | WS_TABSTOP
```

Comment:
Where do these changes come from?

Reply:
From an older branch. Intention is to avoid accelerator collision, CHECKBOX "Dev&re Dışı" vs. CHECKBOX "Gö&rünür". See commits comment.